modules/install-central-operator.adoc

```diff
@@ -37,9 +37,9 @@ spec:
   central:
     declarativeConfiguration:
       configMaps:
-      - name: "declarative-configs"
+      - name: "<declarative-configs>" <1>
       secrets:
-      - name: "sensitive-declarative-configs"
+      - name: "<sensitive-declarative-configs>" <2>
```

Comment:
Also not seeing the changes in the preview here https://docs.openshift.com/acs/4.1/installing/installing_ocp/install-central-ocp.html#install-central-operator_install-central-ocp